config.py

```diff
@@ -133,7 +133,7 @@ class Test(Config):
     JIRA_FEATURES = True
 
     JUST_IN_TIME_ASSESSMENTS = False
-    DM_SEND_EMAIL_TO_STDERR = False
```

Comment:
SEND_EMAILS could be false instead? 
I like that it's testing email rendering, just that test environments can't access amazon SES.